src/Package/Impl/Commands/Markdown/VsMdCommandFactory.cs

```diff
@@ -41,9 +41,9 @@ internal class VsMdCommandFactory : ICommandFactory {
             }
 
             return new ICommand[] {
-                new PreviewHtmlCommand(textView, _workflowProvider, VsAppShell.Current, new ProcessServices(), new FileSystem(), _wbs),
-                new PreviewPdfCommand(textView, _workflowProvider, VsAppShell.Current, new ProcessServices(), new FileSystem()),
-                new PreviewWordCommand(textView, _workflowProvider, VsAppShell.Current, new ProcessServices(), new FileSystem()),
+                new PreviewHtmlCommand(textView, _workflowProvider, VsAppShell.Current),
```

Comment:
After `Services` are introduced, I believe `ICoreShell`, which you can get it from `ImportingConstructor`, should be enough here.